src/axolotl/utils/data/utils.py

```
@@ -9,6 +9,7 @@
 import numpy as np
 import requests
 from datasets import Dataset, IterableDataset
+from typing import Optional
```

Pylint (3.3.7)

[convention] standard import "typing.Optional" should be placed before third party imports "huggingface_hub", "numpy", "requests", "datasets.Dataset"

(C0411)